config/routes.rb

```diff
@@ -68,5 +68,7 @@
   namespace :admin do
     get "/", to: "dashboard#index"
     get "/users", to: "dashboard#users"
+    get "/merchants", to: "merchants#index"
+    patch "/merchants/:merchant_id/disable", to: "merchants#disable"
```

Comment:
Looks like a good move to move away from RESTful methods here, but let's keep this in our minds.